trino/base/hive-config.yaml

```diff
@@ -0,0 +1,189 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hive-config
+data:
+  hive-exec-log4j2.properties: 'status = INFO
+
```

Comment:
You can use Multiline YAML strings here and you don't have to leave the blank lines in between:

```yaml
a: |
  each
  of
  these
  words
  are
  on
  separate
  lines
```

https://yaml-multiline.info/